src/components/EventCardsGridContainer/index.js

```diff
@@ -265,39 +265,43 @@ class EventCardsGrid extends React.Component {
   renderOracles(oracles, eventStatusIndex) {
     const rowItems = [];
     _.each(oracles, (oracle) => {
+      const amount = parseFloat(_.sum(oracle.amounts).toFixed(2));
       let buttonText;
+      let amountLabel;
```

Comment:
we could set the default to 
```
let amountLabel = `${amount} ${oracle.token}`;
```
and only in the Finalize set it to undefined. Save 3 lines :)

Reply:
good call, fixed 👍